projects/packages/jetpack-mu-wpcom/src/features/admin-color-schemes/admin-color-schemes.php

```diff
@@ -0,0 +1,155 @@
+<?php
+/**
+ * Additional admin color schemes.
+ *
+ * The content of this file is mostly copied from projects/plugins/jetpack/modules/masterbar/admin-color-schemes/class-admin-color-schemes.php.
```

Comment:
Can we take this opportunity to move the whole feature to the mu-wpcom plugin? It would make things easier to maintain on the long term I think.

Reply:
I would love to and that's the original plan. However (as noted in the PR description), the color scheme CSS files contain too many things mixed up (e.g. upsell nudge CSS) that are not directly relevant to the current Untangling work scope. Extracting and untangling those CSS, while still making sure that we don't break the nav-unified sites, will take too much time at this point.

I guess we can revisit this once we enable the untangled views to all Simple/Atomic sites. At that point, the original class should not used anymore. Does that sound good?

Reply:
Sounds good to me, thank you!